CHANGELOG.md

```diff
@@ -7,9 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.1.0] - 2023-10-04
+
 ### Added
 
-- `abi` directory to the `@cartesi/rollups` package, ideal for language bindings.
```

Comment:
Did you remove it?

Reply:
Yes, because @gligneul said they are using `export/artifacts` instead to produce Rust bindings.

Reply:
He said it's fine if we keep only distributing `export/artifacts`.